server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -141,6 +143,7 @@
  */
 public class Tablet extends TabletBase {
   private static final Logger log = LoggerFactory.getLogger(Tablet.class);
+  private static final Logger DEDUPE_LOGGER = new DeduplicatingLogger(log, Duration.ofMinutes(5));
```

Comment:
Could name this according to its purpose rather than its impl.  Maybe something like CLOSING_STUCK_LOGGER.

```suggestion
  private static final Logger CLOSING_STUCK_LOGGER = new DeduplicatingLogger(log, Duration.ofMinutes(5));
```